agent/taskresource/cgroup/control/cgroup_controller_linux_test.go

```diff
@@ -44,8 +44,7 @@ func TestCreateHappyCase(t *testing.T) {
 
 	control := newControl(mockCgroupFactory)
 
-	res, err := control.Create(&Spec{testCgroupRoot, testSpecs})
-	assert.Equal(t, mockCgroup, res)
+	err := control.Create(&Spec{testCgroupRoot, testSpecs})
```

Comment:
how are we validating that `Create` is doing what it's supposed to now?

Reply:
`Create` used to return a `cgroups.Cgroup` object that was never actually used in our codebase. I removed it because cgroups v2 doesn't have this object anymore, and since we didn't use it I thought it better to remove than to create an entirely new interface just for cgroupsv2. 

See cgroups.New (v1): https://pkg.go.dev/github.com/containerd/cgroups#New
versus cgroupsv2.NewSystemd (v2): https://pkg.go.dev/github.com/containerd/cgroups@v1.0.3/v2#NewSystemd

Reply:
I see, So if I commented out all the code inside `Create` and just did `return nil`, would any UT break?

Reply:
`mockCgroupFactory.EXPECT().New` would fail